lib/convertkit/client/subscribers.rb

```diff
@@ -9,6 +9,15 @@ def subscriber(subscriber_id)
         connection.get("subscribers/#{subscriber_id}")
       end
 
+      def update_subscriber(subscriber_id, options)
+        response = connection.put("subscribers/#{subscriber_id}") do |f|
+          f.params["email_address"] = options[:email_address]
+          f.params["fields"] = options[:fields]
+          f.params["first_name"] = options[:first_name]
+        end
+        response.body
```

Comment:
I like to return the response body instead of `Faraday::Response` as we do in other endpoints. This creates some inconsistency, but I think that we should return a JSON instead of `Faraday::Response` everywhere.

@rkcudjoe What do you think about it?

Reply:
@mrhead You're right! I've come to realize that while working on the Hubstaff API client and planned to change that here. You should definitely return JSON.